Patches/Forsaken Fauna/Races_BehmothCE.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Forsakens: Fauna</modName>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/ThingDef[defName="FO_RBehemoth"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Birdlike</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="FO_RBehemoth"]/statBases</xpath>
+				<value>
+			<AimingAccuracy>1.0</AimingAccuracy>
+			<ShootingAccuracyPawn>1</ShootingAccuracyPawn>
+					<MeleeDodgeChance>0.03</MeleeDodgeChance>
+					<MeleeCritChance>1.5</MeleeCritChance>
+			<MeleeParryChance>0.4</MeleeParryChance>
+				</value>
+			</li>
+
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="FO_RBehemoth"]/race/baseHealthScale</xpath>
+		<value>
+			<baseHealthScale>6</baseHealthScale>
+		</value>
+	</li>
+
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="FO_RBehemoth"]/statBases/ArmorRating_Blunt</xpath>
+		<value>
+			<ArmorRating_Blunt>11</ArmorRating_Blunt>
+		</value>
+	</li>
+
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="FO_RBehemoth"]/statBases/ArmorRating_Sharp</xpath>
+		<value>
+			<ArmorRating_Sharp>14</ArmorRating_Sharp>
+		</value>
+	</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="FO_RBehemoth"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+				<label>left dragonclaw</label>
+				<capacities>
+					<li>Scratch</li>
+				</capacities>
+							<power>48</power>
+							<cooldownTime>2.58</cooldownTime>
+							<linkedBodyPartsGroup>FrontLeftPaw</linkedBodyPartsGroup>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<armorPenetrationBlunt>25</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+				<label>Right dragonclaw</label>
+				<capacities>
+					<li>Scratch</li>
+				</capacities>
+							<power>48</power>
+							<cooldownTime>2.58</cooldownTime>
+							<linkedBodyPartsGroup>FrontRightPaw</linkedBodyPartsGroup>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<armorPenetrationBlunt>25</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+				<label>razorfangs</label>
+				<capacities>
+					<li>Bite</li>
+				</capacities>
+							<power>88</power>

```

Comment:
This will insta-kill colonists, I'm fairly certain. Is that the intention?

Reply:
Yes, Behemoth is a legendary creature, this is in line with the attack power of other megafauna.